--- dx-4.1.0/configure.in.wiget	Mon Jul 23 18:35:52 2001
+++ dx-4.1.0/configure.in	Mon Jul 23 19:20:03 2001
@@ -224,10 +224,10 @@
 
 dnl Checks for programs.
 AC_PROG_RANLIB
-DX_PROG_CXX()
-DX_PROG_CC
-DX_PROG_CPP
-AC_CYGWIN
+AC_PROG_CXX()
+AC_PROG_CC
+AC_PROG_CPP
+AC_PROG_CXXCPP
 DX_EXEEXT
 if test -z "$EXEEXT" ; then
 	DOT_EXE_EXT="";
@@ -312,15 +312,14 @@
 AC_HEADER_TIME
 AC_STRUCT_TM
 
-if test "$CYGWIN" = yes || test "$MINGW32" = yes ; then
-    ANYDX=AnyDX.dll
-else
-    ANYDX=libAnyDX.so
-fi
+case $host_os in
+	*cygwin*)  ANYDX=AnyDX.dll ;;
+	*mingw32*) ANYDX=AnyDX.dll ;;
+	*)         ANYDX=libAnyDX.so ;;
+esac
 
-AC_LANG_C
+AC_LANG(C)
 
-DX_CYGWIN_MOUNTS
 DX_SET_RTL_FLAGS
 AC_CHECK_LIB(gen, regex)
 
@@ -913,7 +912,7 @@
 
 
 
-DX_CHECK_HEADERS( CC/libc.h CC/osfcn.h DPS/XDPSlib.h DPS/dpsXclient.h \
+AC_CHECK_HEADERS( CC/libc.h CC/osfcn.h DPS/XDPSlib.h DPS/dpsXclient.h \
     DPS/psops.h GL/gl.h GL/glx.h Mrm/MrmPubli.h X11/Composite.h \
     X11/CompositeP.h X11/Constraint.h X11/CoreP.h X11/Intrinsic.h \
     X11/IntrinsicI.h X11/IntrinsicP.h X11/Protocols.h X11/Shell.h \
@@ -952,7 +951,7 @@
 
 DX_STREAM_O2
 
-AC_LANG_CPLUSPLUS
+AC_LANG(C++)
 
 AC_TRY_LINK([
 #include <unistd.h>
--- dx-4.1.0/acinclude.m4.wiget	Mon Jul 23 18:41:34 2001
+++ dx-4.1.0/acinclude.m4	Mon Jul 23 19:23:30 2001
@@ -1,29 +1,15 @@
 dnl
-dnl  Check for the CYGWIN environment
-dnl  -------------------------------------------------------------
-AC_DEFUN(AC_CYGWIN,
-[AC_CACHE_CHECK(for Cygwin environment, ac_cv_cygwin,
-[AC_TRY_COMPILE(,[
-#ifndef __CYGWIN__
-#define __CYGWIN__ __CYGWIN32__
-#endif
-return __CYGWIN__;],
-ac_cv_cygwin=yes, ac_cv_cygwin=no)
-rm -f conftest*])
-CYGWIN=
-test "$ac_cv_cygwin" = yes && CYGWIN=yes])
-
-dnl
 dnl  If using CYGWIN, then the extensions to the filenames need to be
 dnl  different than that of UN*X. This sets that up.
 dnl
 AC_DEFUN(DX_EXEEXT,
-[AC_REQUIRE([AC_CYGWIN])
+[AC_REQUIRE([AC_CANONICAL_HOST])
 AC_MSG_CHECKING([for executable suffix])
 AC_CACHE_VAL(ac_cv_exeext,
-[if test "$CYGWIN" = yes || test "$MINGW32" = yes; then
-  ac_cv_exeext=.exe
-else
+[case $host_os in 
+	*cygwin*)  ac_cv_exeext=.exe ;;
+	*mingw32*) ac_cv_exeext=.exe ;;
+	*)
   rm -f conftest*
   echo 'int main () { return 0; }' > conftest.$ac_ext
   ac_cv_exeext=
@@ -39,7 +25,8 @@
   fi
   rm -f conftest*
   test x"${ac_cv_exeext}" = x && ac_cv_exeext=no
-fi])
+  ;;
+esac])
 EXEEXT=""
 test x"${ac_cv_exeext}" != xno && EXEEXT=${ac_cv_exeext}
 AC_MSG_RESULT(${ac_cv_exeext})
@@ -79,132 +66,6 @@
 
 
 dnl
-dnl  Check the headers for the DX build. This is similar to AC_CHECK_HEADER, but
-dnl  avoids putting every header in the dxconfig.h file.
-dnl  -------------------------------------------------------------
-AC_DEFUN(DX_CHECK_HEADER,
-[dnl Do the transliteration at runtime so arg 1 can be a shell variable.
-ac_safe=`echo "$1" | sed 'y%./+-%__p_%'`
-AC_MSG_CHECKING([for $1])
-AC_CACHE_VAL(ac_cv_header_$ac_safe,
-[DX_TRY_CPP([#include <$1>], eval "ac_cv_header_$ac_safe=yes",
-  eval "ac_cv_header_$ac_safe=no")])dnl
-if eval "test \"`echo '$ac_cv_header_'$ac_safe`\" = yes"; then
-  AC_MSG_RESULT(yes)
-  ifelse([$2], , :, [$2])
-else
-  AC_MSG_RESULT(no)
-ifelse([$3], , , [$3
-])dnl
-fi
-])
-
-
-dnl
-dnl  Check the headers for the DX build. This is similar to AC_CHECK_HEADERS, but
-dnl  avoids putting every header in the dxconfig.h file.
-dnl  -------------------------------------------------------------
-AC_DEFUN(DX_CHECK_HEADERS,
-[for ac_hdr in $1
-do
-DX_CHECK_HEADER($ac_hdr,
-[changequote(, )dnl
-  ac_tr_hdr=HAVE_`echo $ac_hdr | sed 'y%abcdefghijklmnopqrstuvwxyz./-%ABCDEFGHIJKLMNOPQRSTUVWXYZ___%'`
-changequote([, ])dnl
-  AC_DEFINE_UNQUOTED($ac_tr_hdr) $2], $3)dnl
-done
-])
-
-
-dnl
-dnl  Try out if the C++ compiler works
-dnl  DX_TRY_CPP(INCLUDES, [ACTION-IF-TRUE [, ACTION-IF-FALSE]])
-dnl  -------------------------------------------------------------
-AC_DEFUN(DX_TRY_CPP,
-[DX_REQUIRE_CPP()dnl
-cat > conftest.$ac_ext <<EOF
-[#]line __oline__ "configure"
-#include "confdefs.h"
-[$1]
-EOF
-dnl Capture the stderr of cpp.  eval is necessary to expand ac_cpp.
-dnl We used to copy stderr to stdout and capture it in a variable, but
-dnl that breaks under sh -x, which writes compile commands starting
-dnl with ` +' to stderr in eval and subshells.
-ac_try="$ac_cpp conftest.$ac_ext >conftest.out 2>conftest.err"
-AC_TRY_EVAL(ac_try)
-ac_err=`grep -v '^ *+' conftest.err | grep -v "^conftest.${ac_ext}\$"`
-if test -z "$ac_err"; then
-  ifelse([$2], , :, [rm -rf conftest*
-  $2])
-else
-  echo "$ac_err" >&AC_FD_CC
-  echo "configure: failed program was:" >&AC_FD_CC
-  cat conftest.$ac_ext >&AC_FD_CC
-ifelse([$3], , , [  rm -rf conftest*
-  $3
-])dnl
-fi
-rm -f conftest*])
-
-
-dnl
-dnl  Require finding the C or C++ preprocessor, whichever is the
-dnl  current language.
-dnl  -------------------------------------------------------------
-AC_DEFUN(DX_REQUIRE_CPP,
-[ifelse(AC_LANG, C, [AC_REQUIRE([DX_PROG_CPP])], [AC_REQUIRE([DX_PROG_CXXCPP])])])
-
-AC_DEFUN(DX_PROG_CXXCPP,
-[AC_MSG_CHECKING(how to run the C++ preprocessor)
-if test -z "$CXXCPP"; then
-AC_CACHE_VAL(ac_cv_prog_CXXCPP,
-[AC_LANG_SAVE[]dnl
-AC_LANG_CPLUSPLUS[]dnl
-  CXXCPP="${CXX-g++} -E"
-  DX_TRY_CPP([#include <stdlib.h>], , CXXCPP=/lib/cpp)
-  ac_cv_prog_CXXCPP="$CXXCPP"
-AC_LANG_RESTORE[]dnl
-fi])dnl
-CXXCPP="$ac_cv_prog_CXXCPP"
-AC_MSG_RESULT($CXXCPP)
-AC_SUBST(CXXCPP)dnl
-])
-
-AC_DEFUN(DX_PROG_CPP,
-[AC_MSG_CHECKING(how to run the C preprocessor)
-# On Suns, sometimes $CPP names a directory.
-if test -n "$CPP" && test -d "$CPP"; then
-  CPP=
-fi
-if test -z "$CPP"; then
-AC_CACHE_VAL(ac_cv_prog_CPP,
-[  # This must be in double quotes, not single quotes, because CPP may get
-  # substituted into the Makefile and "${CC-cc}" will confuse make.
-  CPP="${CC-cc} -E"
-  # On the NeXT, cc -E runs the code through the compiler's parser,
-  # not just through cpp.
-dnl Use a header file that comes with gcc, so configuring glibc
-dnl with a fresh cross-compiler works.
-  DX_TRY_CPP([#include <assert.h>
-Syntax Error], ,
-  CPP="${CC-cc} -E -traditional-cpp"
-  DX_TRY_CPP([#include <assert.h>
-Syntax Error], ,
-  CPP="${CC-cc} -nologo -E"
-  DX_TRY_CPP([#include <assert.h>
-Syntax Error], , CPP=/lib/cpp)))
-  ac_cv_prog_CPP="$CPP"])dnl
-  CPP="$ac_cv_prog_CPP"
-else
-  ac_cv_prog_CPP="$CPP"
-fi
-AC_MSG_RESULT($CPP)
-AC_SUBST(CPP)dnl
-])
-
-
-dnl
 dnl  Check whether using GNU C
 dnl  -------------------------------------------------------------
 AC_DEFUN(DX_PROG_CC_GNU,
@@ -670,8 +531,8 @@
 dnl  -------------------------------------------------------------
 AC_DEFUN(DX_CYGWIN_MOUNTS,
 [
-    changequote(<<,>>)dnl
     AC_MSG_CHECKING(intelnt under cygwin)
+    changequote(%,%)dnl
     mnts="none"
     if test "$ARCH" = "intelnt" ; then
 	    tt=`uname -s  | tr A-Z a-z | sed "s/^.*cygwin.*$/yes/"`
--- dx-4.1.0/acconfig.h.wiget	Mon Jul 23 20:18:08 2001
+++ dx-4.1.0/acconfig.h	Mon Jul 23 20:18:46 2001
@@ -43,6 +43,7 @@
 #undef HAVE_LIBTIFF
 #undef HAVE_LIBTTF
 #undef HAVE_TT_FONT_PATH
+#undef TT_FONT_PATH
 
 #undef __int64
 
