diff -urN dx-4.1.0/bin/Makefile.am dx-4.1.0-DESTDIR/bin/Makefile.am
--- dx-4.1.0/bin/Makefile.am	Tue Dec 14 05:18:10 1999
+++ dx-4.1.0-DESTDIR/bin/Makefile.am	Sun Aug 27 15:43:37 2000
@@ -1,6 +1,6 @@
 ## Process this file with automake to produce Makefile.in
 
-dxbindir = $(prefix)/dx/bin
+dxbindir = $(prefix)/bin
 bindir = $(prefix)/bin
 
 EXTRA_DIST = dxworker dx mdf2c
diff -urN dx-4.1.0/doc/Makefile.am dx-4.1.0-DESTDIR/doc/Makefile.am
--- dx-4.1.0/doc/Makefile.am	Mon Apr  5 13:57:03 1999
+++ dx-4.1.0-DESTDIR/doc/Makefile.am	Sun Aug 27 17:11:47 2000
@@ -1,6 +1,6 @@
 ## Process this file with automake to produce Makefile.in
 
-docdir = $(prefix)/dx/doc
+docdir = $(prefix)/share/doc/dx
 
 doc_DATA = \
 	README README_SMP README_alphax README_aviion README_hp700 \
diff -urN dx-4.1.0/fonts/Makefile.am dx-4.1.0-DESTDIR/fonts/Makefile.am
--- dx-4.1.0/fonts/Makefile.am	Tue Jun 15 17:54:34 1999
+++ dx-4.1.0-DESTDIR/fonts/Makefile.am	Sun Aug 27 14:30:06 2000
@@ -1,6 +1,6 @@
 ## Process this file with automake to produce Makefile.in
 
-fontdir = $(prefix)/dx/fonts
+fontdir = $(datadir)/dx/fonts
 
 font_DATA = \
 	Readme area.dx cyril_d.dx \
diff -urN dx-4.1.0/help/Makefile.am dx-4.1.0-DESTDIR/help/Makefile.am
--- dx-4.1.0/help/Makefile.am	Wed Feb  2 23:54:07 2000
+++ dx-4.1.0-DESTDIR/help/Makefile.am	Sun Aug 27 14:23:20 2000
@@ -1,6 +1,6 @@
 ## Process this file with automake to produce Makefile.in
 
-helpdir = $(prefix)/dx/help
+helpdir = $(DESTDIR)/$(datadir)/dx/hel
 
 EXTRA_DIST = help.tar
 
diff -urN dx-4.1.0/html/Makefile.am dx-4.1.0-DESTDIR/html/Makefile.am
--- dx-4.1.0/html/Makefile.am	Wed Dec 15 09:07:12 1999
+++ dx-4.1.0-DESTDIR/html/Makefile.am	Sun Aug 27 17:23:35 2000
@@ -1,6 +1,6 @@
 ## Process this file with automake to produce Makefile.in
 
-htmldir = $(prefix)/dx/html
+htmldir = $(prefix)/share/doc/dx/html
 
 html_DATA = README.htm README_SMP.htm README_alphax.htm \
 	README_hp700.htm README_ibm6000.htm README_intelnt.htm \
diff -urN dx-4.1.0/include/Makefile.am dx-4.1.0-DESTDIR/include/Makefile.am
--- dx-4.1.0/include/Makefile.am	Wed May 26 01:35:16 1999
+++ dx-4.1.0-DESTDIR/include/Makefile.am	Sun Aug 27 14:43:31 2000
@@ -1,3 +1,3 @@
-dxincdir = $(prefix)/dx/include
+dxincdir = $(includedir)
 SUBDIRS = dx
 dxinc_HEADERS = dxl.h dxconfig.h
diff -urN dx-4.1.0/include/dx/Makefile.am dx-4.1.0-DESTDIR/include/dx/Makefile.am
--- dx-4.1.0/include/dx/Makefile.am	Wed May 26 01:35:16 1999
+++ dx-4.1.0-DESTDIR/include/dx/Makefile.am	Sun Aug 27 14:43:39 2000
@@ -1,6 +1,6 @@
 ## Process this file with automake to produce Makefile.in
 
-dxincdir = $(prefix)/dx/include/dx
+dxincdir = $(includedir)/dx
  
 dxinc_HEADERS = \
 	UserInteractors.h advanced.h arch.h array.h arrayhandles.h \
diff -urN dx-4.1.0/lib/Makefile.am dx-4.1.0-DESTDIR/lib/Makefile.am
--- dx-4.1.0/lib/Makefile.am	Fri Aug 13 02:15:26 1999
+++ dx-4.1.0-DESTDIR/lib/Makefile.am	Sun Aug 27 15:47:27 2000
@@ -1,5 +1,5 @@
 
-libdir = $(prefix)/dx/lib
 
+libdir	= $(datadir)/dx/lib/
 EXTRA_DIST = mdf2c.awk messages dx.mdf colors.txt dxfSaveCurrentImage.net dxrc dxexec.exp outboard.c
 lib_DATA = $(EXTRA_DIST)
diff -urN dx-4.1.0/man/catl/Makefile.am dx-4.1.0-DESTDIR/man/catl/Makefile.am
--- dx-4.1.0/man/catl/Makefile.am	Mon Apr  5 13:56:00 1999
+++ dx-4.1.0-DESTDIR/man/catl/Makefile.am	Sun Aug 27 14:58:51 2000
@@ -1,6 +1,6 @@
 ## Process this file with automake to produce Makefile.in
 
-catdir = $(prefix)/dx/man/catl
+catdir = $(datadir)/man/catl
 
 cat_DATA = dx.l
 EXTRA_DIST = $(cat_DATA)
diff -urN dx-4.1.0/man/manl/Makefile.am dx-4.1.0-DESTDIR/man/manl/Makefile.am
--- dx-4.1.0/man/manl/Makefile.am	Mon Apr  5 13:55:25 1999
+++ dx-4.1.0-DESTDIR/man/manl/Makefile.am	Sun Aug 27 16:42:52 2000
@@ -1,6 +1,6 @@
 ## Process this file with automake to produce Makefile.in
 
-mandir = $(prefix)/dx/man/manl
+mandir = $(prefix)/man/man1
 
 man_DATA = dx.l
 EXTRA_DIST = $(man_DATA)
diff -urN dx-4.1.0/src/exec/dxexec/Makefile.am dx-4.1.0-DESTDIR/src/exec/dxexec/Makefile.am
--- dx-4.1.0/src/exec/dxexec/Makefile.am	Fri Oct  1 20:42:22 1999
+++ dx-4.1.0-DESTDIR/src/exec/dxexec/Makefile.am	Sun Aug 27 15:03:49 2000
@@ -5,7 +5,7 @@
 EXPARG=@EXPARG@
 EXP=$BUILDBASE/lib/dxexec.exp
 
-dxbindir = $(prefix)/dx/bin_@ARCH@
+dxbindir = $(datadir)/dx/bin_@ARCH@
 dxbin_PROGRAMS = dxexec
 noinst_LIBRARIES = libDXEXEC.a
 
diff -urN dx-4.1.0/src/exec/dxmods/Makefile.am dx-4.1.0-DESTDIR/src/exec/dxmods/Makefile.am
--- dx-4.1.0/src/exec/dxmods/Makefile.am	Thu Jan  6 22:06:54 2000
+++ dx-4.1.0-DESTDIR/src/exec/dxmods/Makefile.am	Sun Aug 27 15:12:35 2000
@@ -2,7 +2,7 @@
 
 INCLUDES = -I../../../include -I../dpexec -I/usr/include -I/usr/local/include -D@ARCH@
 
-dxmdfdir = $(prefix)/dx/lib
+dxmdfdir = $(datadir)/lib/
 noinst_LIBRARIES = libDXMODS.a libusercm.a
 SUFFIXES = .c .y .c .s .h .o 
 
diff -urN dx-4.1.0/src/exec/libdx/Makefile.am dx-4.1.0-DESTDIR/src/exec/libdx/Makefile.am
--- dx-4.1.0/src/exec/libdx/Makefile.am	Wed Dec 15 07:34:46 1999
+++ dx-4.1.0-DESTDIR/src/exec/libdx/Makefile.am	Sun Aug 27 15:03:21 2000
@@ -2,7 +2,7 @@
 
 INCLUDES = -I../../../include  -D@ARCH@
 
-dxlibdir = $(prefix)/dx/lib_@ARCH@
+dxlibdir = $(datadir)/dx/lib_@ARCH@
 
 noinst_LIBRARIES = libLIBDX.a libcallm.a
 dxlib_LIBRARIES = libDXlite.a 
diff -urN dx-4.1.0/src/misc/Makefile.am dx-4.1.0-DESTDIR/src/misc/Makefile.am
--- dx-4.1.0/src/misc/Makefile.am	Mon Jan 17 23:52:31 2000
+++ dx-4.1.0-DESTDIR/src/misc/Makefile.am	Sun Aug 27 15:08:31 2000
@@ -4,7 +4,7 @@
 
 dxbindir = $(prefix)/bin
 noinst_PROGRAMS = dxlocal
-archmakdir = $(prefix)/dx/lib_$(ARCH)
+archmakdir = $(datadir)/dx/lib_$(ARCH)
 archmak_DATA = arch.mak
 
 dxlocal_SOURCES = dx.c 
diff -urN dx-4.1.0/src/uipp/dxl/Makefile.am dx-4.1.0-DESTDIR/src/uipp/dxl/Makefile.am
--- dx-4.1.0/src/uipp/dxl/Makefile.am	Mon May  3 16:08:43 1999
+++ dx-4.1.0-DESTDIR/src/uipp/dxl/Makefile.am	Sun Aug 27 15:46:58 2000
@@ -2,7 +2,7 @@
 
 INCLUDES = -I../../../include  -D@ARCH@
 
-dxlibdir = $(prefix)/dx/lib_@ARCH@
+dxlibdir = $(prefix)/lib
 
 dxlib_LIBRARIES = libDXL.a
 
diff -urN dx-4.1.0/src/uipp/dxui/Makefile.am dx-4.1.0-DESTDIR/src/uipp/dxui/Makefile.am
--- dx-4.1.0/src/uipp/dxui/Makefile.am	Wed Sep  1 23:27:44 1999
+++ dx-4.1.0-DESTDIR/src/uipp/dxui/Makefile.am	Sun Aug 27 15:46:09 2000
@@ -2,7 +2,7 @@
 
 INCLUDES = -I../base -I../../../include -I../dxuilib -D@ARCH@
 
-dxbindir = $(prefix)/dx/bin_@ARCH@
+dxbindir = $(prefix)/bin
 dxbin_PROGRAMS = dxui
 
 
diff -urN dx-4.1.0/src/uipp/mb/Makefile.am dx-4.1.0-DESTDIR/src/uipp/mb/Makefile.am
--- dx-4.1.0/src/uipp/mb/Makefile.am	Mon May  3 16:08:58 1999
+++ dx-4.1.0-DESTDIR/src/uipp/mb/Makefile.am	Sun Aug 27 15:44:20 2000
@@ -2,7 +2,7 @@
 
 INCLUDES = -I../../../include -I../base -I../widgets -D@ARCH@
 
-dxbindir = $(prefix)/dx/bin_@ARCH@
+dxbindir = $(prefix)/bin
 
 dxbin_PROGRAMS = builder
 
diff -urN dx-4.1.0/src/uipp/prompter/Makefile.am dx-4.1.0-DESTDIR/src/uipp/prompter/Makefile.am
--- dx-4.1.0/src/uipp/prompter/Makefile.am	Mon May  3 16:08:59 1999
+++ dx-4.1.0-DESTDIR/src/uipp/prompter/Makefile.am	Sun Aug 27 15:44:53 2000
@@ -2,7 +2,7 @@
 
 INCLUDES =  -I../base -I../../../include -D@ARCH@
 
-dxbindir = $(prefix)/dx/bin_@ARCH@
+dxbindir = $(prefix)/bin
 
 dxbin_PROGRAMS = prompter
 
diff -urN dx-4.1.0/src/uipp/startup/Makefile.am dx-4.1.0-DESTDIR/src/uipp/startup/Makefile.am
--- dx-4.1.0/src/uipp/startup/Makefile.am	Mon May  3 16:08:59 1999
+++ dx-4.1.0-DESTDIR/src/uipp/startup/Makefile.am	Sun Aug 27 15:45:12 2000
@@ -2,7 +2,7 @@
 
 INCLUDES =  -I../../../include -I../base -D@ARCH@
 
-dxbindir = $(prefix)/dx/bin_@ARCH@
+dxbindir = $(prefix)/bin
 
 dxbin_PROGRAMS = startupui
 
diff -urN dx-4.1.0/src/uipp/tutor/Makefile.am dx-4.1.0-DESTDIR/src/uipp/tutor/Makefile.am
--- dx-4.1.0/src/uipp/tutor/Makefile.am	Mon May  3 16:09:00 1999
+++ dx-4.1.0-DESTDIR/src/uipp/tutor/Makefile.am	Sun Aug 27 15:45:36 2000
@@ -2,7 +2,7 @@
 
 INCLUDES = -I../../../include -I../base -D@ARCH@
 
-dxbindir = $(prefix)/dx/bin_@ARCH@
+dxbindir = $(prefix)/bin
 
 dxbin_PROGRAMS = tutor
 
diff -urN dx-4.1.0/src/uipp/ui/Makefile.am dx-4.1.0-DESTDIR/src/uipp/ui/Makefile.am
--- dx-4.1.0/src/uipp/ui/Makefile.am	Mon Jun  7 21:19:27 1999
+++ dx-4.1.0-DESTDIR/src/uipp/ui/Makefile.am	Sun Aug 27 15:02:30 2000
@@ -1,6 +1,6 @@
 ## Process this file with automake to produce Makefile.in
 
-uidatadir = $(prefix)/dx/ui
+uidatadir = $(datadir)/dx/ui
 
 DATA1 = \
 	icon50.dat logo.dat ui.mdf Basic2D.cfg Basic2D.net \
