diff -urN dx-4.3.0.orig/bin/Makefile.am dx-4.3.0/bin/Makefile.am
--- dx-4.3.0.orig/bin/Makefile.am	2003-07-11 07:41:19.000000000 +0200
+++ dx-4.3.0/bin/Makefile.am	2003-08-09 07:04:25.000000000 +0200
@@ -1,6 +1,6 @@
 ## Process this file with automake to produce Makefile.in
 
-dxbindir = $(prefix)/dx/bin
+dxbindir = $(prefix)/bin
 bindir = $(prefix)/bin
 
 EXTRA_SCRIPTS = mdf2c dx
diff -urN dx-4.3.0.orig/doc/Makefile.am dx-4.3.0/doc/Makefile.am
--- dx-4.3.0.orig/doc/Makefile.am	1999-04-05 13:57:03.000000000 +0200
+++ dx-4.3.0/doc/Makefile.am	2003-08-09 07:04:48.000000000 +0200
@@ -1,6 +1,6 @@
 ## Process this file with automake to produce Makefile.in
 
-docdir = $(prefix)/dx/doc
+docdir = $(datadir)/doc/dx
 
 doc_DATA = \
 	README README_SMP README_alphax README_aviion README_hp700 \
diff -urN dx-4.3.0.orig/fonts/Makefile.am dx-4.3.0/fonts/Makefile.am
--- dx-4.3.0.orig/fonts/Makefile.am	2003-07-11 07:41:19.000000000 +0200
+++ dx-4.3.0/fonts/Makefile.am	2003-08-09 07:04:58.000000000 +0200
@@ -1,6 +1,6 @@
 ## Process this file with automake to produce Makefile.in
 
-fontdir = $(prefix)/dx/fonts
+fontdir = $(datadir)/dx/fonts
 
 font_DATA = \
 	README area.dx cyril_d.dx \
diff -urN dx-4.3.0.orig/help/Makefile.am dx-4.3.0/help/Makefile.am
--- dx-4.3.0.orig/help/Makefile.am	2003-07-11 07:41:19.000000000 +0200
+++ dx-4.3.0/help/Makefile.am	2003-08-09 07:05:13.000000000 +0200
@@ -1,6 +1,6 @@
 ## Process this file with automake to produce Makefile.in
 
-helpdir = $(prefix)/dx/help
+helpdir = $(datadir)/dx/help
 
 EXTRA_DIST = $(helpfiles1) $(helpfiles2) $(helpfiles3) $(helpfiles4) $(helpfiles5)  \
 	$(helpfiles6) $(helpfiles7) $(helpfiles8) $(helpfiles9) $(helpfiles10)  \
diff -urN dx-4.3.0.orig/html/images/Makefile.am dx-4.3.0/html/images/Makefile.am
--- dx-4.3.0.orig/html/images/Makefile.am	2002-04-01 11:04:03.000000000 +0200
+++ dx-4.3.0/html/images/Makefile.am	2003-08-09 07:05:59.000000000 +0200
@@ -1,5 +1,5 @@
 
-imagesdir = $(prefix)/dx/html/images
+imagesdir = $(datadir)/doc/dx/html/images
 
 images1 = \
 	3dcurs.gif adctlpt.gif arch.gif autoax.gif autoexp.gif		\
diff -urN dx-4.3.0.orig/html/Makefile.am dx-4.3.0/html/Makefile.am
--- dx-4.3.0.orig/html/Makefile.am	2003-07-11 07:41:19.000000000 +0200
+++ dx-4.3.0/html/Makefile.am	2003-08-09 07:05:47.000000000 +0200
@@ -1,6 +1,6 @@
 ## Process this file with automake to produce Makefile.in
 
-htmldir = $(prefix)/dx/html
+htmldir = $(datadir)/doc/dx/html
 
 SUBDIRS = pages images
 
diff -urN dx-4.3.0.orig/html/pages/Makefile.am dx-4.3.0/html/pages/Makefile.am
--- dx-4.3.0.orig/html/pages/Makefile.am	2002-06-20 06:40:11.000000000 +0200
+++ dx-4.3.0/html/pages/Makefile.am	2003-08-09 07:06:05.000000000 +0200
@@ -1,4 +1,4 @@
-pagesdir = $(prefix)/dx/html/pages
+pagesdir = $(datadir)/doc/dx/html/pages
 
 pages1 = \
 	insgu002.htm insgu003.htm insgu004.htm insgu005.htm insgu006.htm \
diff -urN dx-4.3.0.orig/include/dx/Makefile.am dx-4.3.0/include/dx/Makefile.am
--- dx-4.3.0.orig/include/dx/Makefile.am	1999-05-26 01:35:16.000000000 +0200
+++ dx-4.3.0/include/dx/Makefile.am	2003-08-09 07:06:30.000000000 +0200
@@ -1,6 +1,6 @@
 ## Process this file with automake to produce Makefile.in
 
-dxincdir = $(prefix)/dx/include/dx
+dxincdir = $(prefix)/include/dx
  
 dxinc_HEADERS = \
 	UserInteractors.h advanced.h arch.h array.h arrayhandles.h \
diff -urN dx-4.3.0.orig/include/Makefile.am dx-4.3.0/include/Makefile.am
--- dx-4.3.0.orig/include/Makefile.am	2001-06-29 15:08:15.000000000 +0200
+++ dx-4.3.0/include/Makefile.am	2003-08-09 07:06:19.000000000 +0200
@@ -1,3 +1,3 @@
-dxincdir = $(prefix)/dx/include
+dxincdir = $(prefix)/include/dx
 SUBDIRS = dx
 dxinc_HEADERS = dxl.h dxconfig.h dxstereo.h
diff -urN dx-4.3.0.orig/lib/Makefile.am dx-4.3.0/lib/Makefile.am
--- dx-4.3.0.orig/lib/Makefile.am	2003-07-11 07:41:54.000000000 +0200
+++ dx-4.3.0/lib/Makefile.am	2003-08-09 07:06:44.000000000 +0200
@@ -1,4 +1,4 @@
-libsdir = $(prefix)/dx/lib
+libsdir = $(prefix)/lib/dx
 
 libs_DATA = colors.txt dxexec.def dxexec.exp dxexec.ifs dxfSaveCurrentImage.net \
 	dxrc mdf2c.awk messages outboard.c
diff -urN dx-4.3.0.orig/man/catl/Makefile.am dx-4.3.0/man/catl/Makefile.am
--- dx-4.3.0.orig/man/catl/Makefile.am	1999-04-05 13:56:00.000000000 +0200
+++ dx-4.3.0/man/catl/Makefile.am	2003-08-09 07:07:01.000000000 +0200
@@ -1,6 +1,6 @@
 ## Process this file with automake to produce Makefile.in
 
-catdir = $(prefix)/dx/man/catl
+catdir = $(datadir)/man/catl
 
 cat_DATA = dx.l
 EXTRA_DIST = $(cat_DATA)
diff -urN dx-4.3.0.orig/man/manl/Makefile.am dx-4.3.0/man/manl/Makefile.am
--- dx-4.3.0.orig/man/manl/Makefile.am	2002-05-20 06:49:40.000000000 +0200
+++ dx-4.3.0/man/manl/Makefile.am	2003-08-09 07:07:13.000000000 +0200
@@ -1,6 +1,6 @@
 ## Process this file with automake to produce Makefile.in
 
-mandir = $(prefix)/dx/man
+mandir = $(datadir)/man
 
 man_MANS = dx.l
 EXTRA_DIST = $(man_MANS)
diff -urN dx-4.3.0.orig/src/exec/dxexec/Makefile.am dx-4.3.0/src/exec/dxexec/Makefile.am
--- dx-4.3.0.orig/src/exec/dxexec/Makefile.am	2003-07-11 07:41:55.000000000 +0200
+++ dx-4.3.0/src/exec/dxexec/Makefile.am	2003-08-09 07:08:13.000000000 +0200
@@ -9,7 +9,7 @@
 libDXEXEC_la_SOURCES = main.c
 libDXEXEC_la_CFLAGS = $(AM_CFLAGS)
 
-bindir = $(prefix)/dx/bin_@ARCH@
+bindir = $(prefix)/bin
 bin_PROGRAMS = dxexec
 
 dxexec_LDADD = ../dpexec/libDPEXEC.la ../dxmods/libDXMODS.la ../dxmods/user.lo \
diff -urN dx-4.3.0.orig/src/exec/dxmods/Makefile.am dx-4.3.0/src/exec/dxmods/Makefile.am
--- dx-4.3.0.orig/src/exec/dxmods/Makefile.am	2003-07-11 07:41:55.000000000 +0200
+++ dx-4.3.0/src/exec/dxmods/Makefile.am	2003-08-09 07:08:26.000000000 +0200
@@ -2,7 +2,7 @@
 
 INCLUDES = -I${top_srcdir}/include -I${srcdir}/../dpexec -D@ARCH@
 
-dxmdfdir = $(prefix)/dx/lib
+dxmdfdir = $(prefix)/lib/dx
 noinst_LTLIBRARIES = libDXMODS.la libusercm.la libuser.la libDXMODSN.la
 SUFFIXES = .c .y .c .s .h .o 
 
diff -urN dx-4.3.0.orig/src/exec/libdx/Makefile.am dx-4.3.0/src/exec/libdx/Makefile.am
--- dx-4.3.0.orig/src/exec/libdx/Makefile.am	2003-07-11 07:42:09.000000000 +0200
+++ dx-4.3.0/src/exec/libdx/Makefile.am	2003-08-09 07:09:32.000000000 +0200
@@ -7,7 +7,7 @@
 #INCLUDES = -I../../../include  -D@ARCH@
 INCLUDES = -I${top_srcdir}/include  -D@ARCH@
 
-dxlibdir = $(prefix)/dx/lib_@ARCH@
+dxlibdir = $(prefix)/lib/dx
 
 noinst_LTLIBRARIES = libLIBDX.la libcallm.la libmem.la
 #dxlib_LIBRARIES = libDXlite.a 
diff -urN dx-4.3.0.orig/src/exec/Makefile.am dx-4.3.0/src/exec/Makefile.am
--- dx-4.3.0.orig/src/exec/Makefile.am	2003-07-11 07:41:54.000000000 +0200
+++ dx-4.3.0/src/exec/Makefile.am	2003-08-09 07:09:27.000000000 +0200
@@ -1,6 +1,6 @@
 SUBDIRS = libdx dxmods hwrender dpexec dxexec
 
-dxlibdir = $(prefix)/dx/lib_@ARCH@
+dxlibdir = $(prefix)/lib/dx
 
 MAJOR_VERSION=@MAJOR_VERSION@
 MINOR_VERSION=@MINOR_VERSION@
diff -urN dx-4.3.0.orig/src/misc/Makefile.am dx-4.3.0/src/misc/Makefile.am
--- dx-4.3.0.orig/src/misc/Makefile.am	2003-07-11 07:42:21.000000000 +0200
+++ dx-4.3.0/src/misc/Makefile.am	2003-08-09 07:10:02.000000000 +0200
@@ -4,7 +4,7 @@
 
 dxbindir = $(prefix)/bin
 noinst_PROGRAMS = dxlocal
-archmakdir = $(prefix)/dx/lib_$(ARCH)
+archmakdir = $(prefix)/lib/dx
 archmak_DATA = arch.mak
 
 dxlocal_SOURCES = dx.c utils.c
diff -urN dx-4.3.0.orig/src/uipp/dxl/Makefile.am dx-4.3.0/src/uipp/dxl/Makefile.am
--- dx-4.3.0.orig/src/uipp/dxl/Makefile.am	2003-07-11 07:42:21.000000000 +0200
+++ dx-4.3.0/src/uipp/dxl/Makefile.am	2003-08-09 07:10:34.000000000 +0200
@@ -7,7 +7,7 @@
 
 INCLUDES = -I${top_srcdir}/include  -D@ARCH@
 
-dxlibdir = $(prefix)/dx/lib_@ARCH@
+dxlibdir = $(prefix)/lib/dx
 
 noinst_HEADERS = \
 	dxlP.h \
diff -urN dx-4.3.0.orig/src/uipp/dxui/Makefile.am dx-4.3.0/src/uipp/dxui/Makefile.am
--- dx-4.3.0.orig/src/uipp/dxui/Makefile.am	2003-02-24 19:34:22.000000000 +0100
+++ dx-4.3.0/src/uipp/dxui/Makefile.am	2003-08-09 07:11:17.000000000 +0200
@@ -2,7 +2,7 @@
 
 INCLUDES = -I${srcdir}/../base -I${top_srcdir}/include -I${srcdir}/../dxuilib -D@ARCH@
 
-dxbindir = $(prefix)/dx/bin_@ARCH@
+dxbindir = $(prefix)/bin
 dxbin_PROGRAMS = dxui
 
 
diff -urN dx-4.3.0.orig/src/uipp/java/Makefile.am dx-4.3.0/src/uipp/java/Makefile.am
--- dx-4.3.0.orig/src/uipp/java/Makefile.am	2003-07-11 07:42:22.000000000 +0200
+++ dx-4.3.0/src/uipp/java/Makefile.am	2003-08-09 07:11:45.000000000 +0200
@@ -27,12 +27,12 @@
 ## libtool and the -shrext flag to make this much more 
 ## portable.
 
-javadxdir = $(prefix)/dx/java/htmlpages
+javadxdir = $(datadir)/dx/java/htmlpages
 javadx_DATA = dx.jar 
-javasrvdir = $(prefix)/dx/java/server/class
+javasrvdir = $(datadir)/dx/java/server/class
 javasrv_DATA = server.jar 
 EXTRA_DIST = $(DX_SRCS) DXServer.java makeall.java makewrl.java exports.awk README
-javalibdir = $(prefix)/dx/java/server/lib_@ARCH@
+javalibdir = $(datadir)/dx/java/server/lib
 
 javalib_DATA = @ANYDX@
 SUBDIRS = server dx
diff -urN dx-4.3.0.orig/src/uipp/java/server/macros/Makefile.am dx-4.3.0/src/uipp/java/server/macros/Makefile.am
--- dx-4.3.0.orig/src/uipp/java/server/macros/Makefile.am	1999-07-09 21:25:13.000000000 +0200
+++ dx-4.3.0/src/uipp/java/server/macros/Makefile.am	2003-08-09 07:12:19.000000000 +0200
@@ -1,6 +1,6 @@
 ##  Process this file with automake to produce Makefile.in
 
-serverdatadir = $(prefix)/dx/java/server/dxmacros
+serverdatadir = $(datadir)/dx/java/server/dxmacros
 
 serverdata_DATA = $(MACRO_FILES)
 
diff -urN dx-4.3.0.orig/src/uipp/java/server/Makefile.am dx-4.3.0/src/uipp/java/server/Makefile.am
--- dx-4.3.0.orig/src/uipp/java/server/Makefile.am	2001-05-31 16:50:15.000000000 +0200
+++ dx-4.3.0/src/uipp/java/server/Makefile.am	2003-08-09 07:12:02.000000000 +0200
@@ -1,6 +1,6 @@
-srvbindir = $(prefix)/dx/java/server/bin
-srvoutputdir = $(prefix)/dx/java/output
-userhtmldir = $(prefix)/dx/java/user
+srvbindir = $(datadir)/dx/java/server/bin
+srvoutputdir = $(datadir)/dx/java/output
+userhtmldir = $(datadir)/dx/java/user
 # the server script (in srvbindir) will actually cd and run from the class subdir so that java will find the classes
 # the server will expect to find a file dxserver.paths there to tell the exec where to find .net files
 srvrundir = $(srvbindir)/../class
diff -urN dx-4.3.0.orig/src/uipp/mb/Makefile.am dx-4.3.0/src/uipp/mb/Makefile.am
--- dx-4.3.0.orig/src/uipp/mb/Makefile.am	2003-07-11 07:42:22.000000000 +0200
+++ dx-4.3.0/src/uipp/mb/Makefile.am	2003-08-09 07:12:50.000000000 +0200
@@ -2,7 +2,7 @@
 
 INCLUDES = -I${top_srcdir}/include -I${srcdir}/../base -I${srcdir}/../widgets -D@ARCH@
 
-dxbindir = $(prefix)/dx/bin_@ARCH@
+dxbindir = $(prefix)/bin
 
 dxbin_PROGRAMS = builder
 
diff -urN dx-4.3.0.orig/src/uipp/prompter/Makefile.am dx-4.3.0/src/uipp/prompter/Makefile.am
--- dx-4.3.0.orig/src/uipp/prompter/Makefile.am	2003-07-11 07:42:23.000000000 +0200
+++ dx-4.3.0/src/uipp/prompter/Makefile.am	2003-08-09 07:13:04.000000000 +0200
@@ -2,7 +2,7 @@
 
 INCLUDES =  -I${srcdir}/../base -I${top_srcdir}/include -D@ARCH@
 
-dxbindir = $(prefix)/dx/bin_@ARCH@
+dxbindir = $(prefix)/bin
 
 dxbin_PROGRAMS = prompter
 
diff -urN dx-4.3.0.orig/src/uipp/startup/Makefile.am dx-4.3.0/src/uipp/startup/Makefile.am
--- dx-4.3.0.orig/src/uipp/startup/Makefile.am	2003-07-11 07:42:23.000000000 +0200
+++ dx-4.3.0/src/uipp/startup/Makefile.am	2003-08-09 07:13:17.000000000 +0200
@@ -2,7 +2,7 @@
 
 INCLUDES =  -I${top_srcdir}/include -I${srcdir}/../base -D@ARCH@
 
-dxbindir = $(prefix)/dx/bin_@ARCH@
+dxbindir = $(prefix)/bin
 
 dxbin_PROGRAMS = startupui
 
diff -urN dx-4.3.0.orig/src/uipp/tutor/Makefile.am dx-4.3.0/src/uipp/tutor/Makefile.am
--- dx-4.3.0.orig/src/uipp/tutor/Makefile.am	2003-07-11 07:42:23.000000000 +0200
+++ dx-4.3.0/src/uipp/tutor/Makefile.am	2003-08-09 07:13:26.000000000 +0200
@@ -2,7 +2,7 @@
 
 INCLUDES = -I${top_srcdir}/include -I${srcdir}/../base -D@ARCH@
 
-dxbindir = $(prefix)/dx/bin_@ARCH@
+dxbindir = $(prefix)/bin
 
 dxbin_PROGRAMS = tutor
 
diff -urN dx-4.3.0.orig/src/uipp/ui/Makefile.am dx-4.3.0/src/uipp/ui/Makefile.am
--- dx-4.3.0.orig/src/uipp/ui/Makefile.am	2001-04-12 16:29:02.000000000 +0200
+++ dx-4.3.0/src/uipp/ui/Makefile.am	2003-08-09 07:13:48.000000000 +0200
@@ -1,6 +1,6 @@
 ## Process this file with automake to produce Makefile.in
 
-uidatadir = $(prefix)/dx/ui
+uidatadir = $(datadir)/dx/ui
 
 DATA1 = \
 	icon50.dat icon50.xpm logo.dat logo.xpm ui.mdf Basic2D.cfg Basic2D.net \
