--- dx-4.3.2/configure.ac.orig	2003-09-16 21:48:18.000000000 +0200
+++ dx-4.3.2/configure.ac	2004-06-21 19:05:31.447745439 +0200
@@ -535,7 +535,6 @@
 fi
 
 dnl A couple libs for cygwin
-AC_CHECK_LIB(REXP, regcomp)
 DX_CHECK_TGMATH
 
 dnl If buildlibdx off then pass this down to the Makefile
@@ -546,7 +545,6 @@
 dnl Check for ImageMagick Libraries
 
 dnl Checks for libraries.
-AC_CHECK_LIB(g++, main)
 AC_CHECK_LIB(dl, main)
 
 dnl Set the CXXFLAG for new keylayout
@@ -1096,11 +1094,10 @@
 AC_CHECK_HEADERS(sstream strstream.h strstrea.h sysent.h)
 AC_CHECK_HEADERS(fstream fstream.h)
 
-AC_CHECK_LIB(socket, socket)
-AC_CHECK_LIB(nsl, inet_addr)
-AC_CHECK_LIB(gen, regex)
-AC_CHECK_LIB(regex, regcomp)
-AC_CHECK_LIB(REXP, regcomp)
+AC_SEARCH_LIBS(socket, socket)
+AC_SEARCH_LIBS(inet_addr, nsl)
+AC_SEARCH_LIBS(regex, gen)
+AC_SEARCH_LIBS(regcomp, [regex REXP])
 
 DX_STREAM_O2
 
