--- dx-4.4.4/configure.ac.orig	2006-08-21 23:56:08.000000000 +0200
+++ dx-4.4.4/configure.ac	2023-11-27 21:22:51.836489093 +0100
@@ -546,7 +546,6 @@ if test "$ARCH" = "solaris" ; then
 fi
 
 dnl A couple libs for cygwin
-AC_CHECK_LIB(REXP, regcomp)
 DX_CHECK_TGMATH
 
 dnl If buildlibdx off then pass this down to the Makefile
@@ -557,7 +556,6 @@ AM_CONDITIONAL(WITH_INSTALLHTML, test $w
 dnl Check for ImageMagick Libraries
 
 dnl Checks for libraries.
-AC_CHECK_LIB(g++, main)
 AC_CHECK_LIB(dl, main)
 
 dnl Set the CXXFLAG for new keylayout
@@ -1157,11 +1155,10 @@ AC_CHECK_HEADERS([regexp.h], [], [],
    #define ERROR(val)  /**/
 ])
 
-AC_CHECK_LIB(socket, socket)
-AC_CHECK_LIB(nsl, inet_addr)
-AC_CHECK_LIB(gen, regex)
-AC_CHECK_LIB(regex, regcomp)
-AC_CHECK_LIB(REXP, regcomp)
+AC_SEARCH_LIBS(socket, socket)
+AC_SEARCH_LIBS(inet_addr, nsl)
+AC_SEARCH_LIBS(regex, gen)
+AC_SEARCH_LIBS(regcomp, [regex REXP])
 
 DX_STREAM_O2
 
